data/map systems.txt

```diff
@@ -5984,6 +6157,67 @@ system Atik
 		distance 1905.05
 		period 270.619
 
+system Atlas
+	pos -145.13889 174.86111
+	government Syndicate
+	link Achernar
+	link "Al Dhanab"
+	link Alpheratz
+	link Meissa
+	link Oneldiam
+	asteroids "small rock" 17 3.069
+	asteroids "medium rock" 5 5.3568
+	asteroids "large rock" 8 4.5756
+	asteroids "small metal" 29 4.5756
+	asteroids "medium metal" 43 4.743
+	asteroids "large metal" 1 2.9016
+	minables silver 24 8.125609
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Large Core Merchants" 500
	fleet "Large Syndicate" 700
	fleet "Small Syndicate" 600
	fleet "Small Core Merchants" 400
	fleet "Republic Logistics" 5800
	fleet "Small Core Pirates" 2000
	fleet "Large Core Pirates" 3000
```
This one is the shortest link between three different regions of the Syndicate, including Achernar, which is the home of Syndicated Shipyards and probably the second most important system in Syndicate space, second only to Markab, so there should be a fairly high frequency of Syndicate fleets here.